package/gluon-mesh-babel/files/lib/gluon/state/check.d/has_neighbours

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+out=$(echo dump | nc ::1 33123 | grep "add neighbour" 2>/dev/null) && [ -n "$out" ]
```

Comment:
```suggestion
echo dump | nc ::1 33123 | grep -qF 'add neighbour'
```
As we're only interested whether there is matching output or not, `grep -q` should be used. Adding `-F` makes it more efficient, as it doesn't interpret the pattern as a regex.

Reply:
I still lack a node/network to test this on;
One possible issue I see are possible error-messages of nc.
I'll apply the commit in a moment. Afterwords this can be marked as blocked by #2353 .